tools/oca_sync_users.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""
+Create res.users for OCA members with a github login filled in.
+
+This enables adding them to project teams in the OCA instance.
+"""
+from __future__ import absolute_import, print_function
+
+import xmlrpclib
+
+from . odoo_login import login, get_parser
+
+
+def main():
+    parser = get_parser(with_help=True)
+    args = parser.parse_args()
+    client = login(args.username, args.store)
+    ResPartner = client.ResPartner
+    ResUsers = client.ResUsers
+    ResGroups = client.ResGroups
+    grp_project_user = ResGroups.get('project.group_project_user')
+    members_with_gh = ResPartner.search([('x_github_login', '!=', False),
+                                         ('user_ids', '=', False)])
```

Comment:
Maybe check for active_test here, so that we don't try to recreate users which have been set to inactive for some reason?